### PR TITLE
Add more login functions

### PR DESCRIPTION
**Pull request by Drakulix**
Hey there, I've been using your library to implement [`logind` support](https://github.com/Smithay/smithay/pull/63) in our [wayland compositor library](https://github.com/Smithay/smithay) and I need those two additional functions to correctly initialize the logind session. (The code is tested and working.)

I hope this PR can be merged as is and released on crates.io soonish, but in case there is anything wrong with my approach, please let me know and I will try to fix this asap.

Greetings,
Victor

### CODE REVIEW

**Comment by Drakulix** [comment]
Rebased on latest master

File: src/login.rs [hunk 1]
```diff
@@ -94,6 +95,29 @@ pub fn get_session(pid: Option<pid_t>) -> Result<String> {
     Ok(ss.to_string())
 }
 
+/// Determines the seat identifier of the seat the session identified
+/// by the specified session identifier belongs to.
+///
+/// Note that not all sessions are attached to a seat, this call will fail for them.
+pub fn get_seat<T: Into<Vec<u8>>>(session: T) -> Result<String> {
```
Comment (jmesmon) [inline src/login.rs]: Could you use `CStringArgument` in place of `Into<Vec<u8>>` here? See https://github.com/jmesmon/rust-systemd/blob/master/src/daemon.rs#L65 for existing usage.

Allows folks to pass `CStrs` directly if they so desire.

**Comment by Drakulix** [comment]
Done. Would you like me to squash the commits?

**Comment by jmesmon** [comment]
Nah, I think it's fine. Thank's for the changes.

**Comment by Drakulix** [comment]
No problem. Could you do a crates.io release with these changes? That would be very helpful. thanks :)

**Comment by jmesmon** [comment]
@Drakulix just did a release with these changes https://crates.io/crates/systemd/0.2.0

**Pull request merged**